buffalo/cmd/build/builder.go

```diff
@@ -38,6 +38,12 @@ func New(ctx context.Context, opts Options) *Builder {
 		b.buildBin,
 	}
 
+	if b.Options.SkipAssets {
```

Comment:
This won't work correctly. If you do this here then the templates, and anything else that uses `packr` won't run. The correct thing to do is to put the logic in the `buildAssets` method.

Reply:
I see, ok, i changed the code to do the check inside the buildAssets function, LMK if it makes sense.